src/components/Spinner/Spinner.stories.mdx

```diff
@@ -17,15 +17,15 @@ This is a [React](https://reactjs.org/) component for the Vanilla [Spin](https:/
 ### Default
 
 <Canvas>
-  <Story name="Default">
+  <Story name="Default" args={{ role: "alert", ariaLive: "polite"}}>
     {Template.bind({})}
   </Story>
 </Canvas>
 
 ### Spinner with text
 
 <Canvas>
-  <Story name="Text" args={{ text: "Loading..." }}>
+  <Story name="Text" args={{ text: "Loading...", role: "alert", ariaLive: "polite" }}>
```

Comment:
This also seems to repeat default values?

Instead of adding these props to existing examples maybe worth adding new example with some accessibility changes? For example how to change role and ariaLive to different values (not sure what different role would be though).

Reply:
Ahh yes I forgot to remove these. I'll add an example with aria-live="assertive"